src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import "./Header.css";
```

Comment:
First, great work having each component with its own CSS file. Makes it modularized 🙌🏾

However, we still run into the issue of global classes. E.g if you have different components with the same class names, the styles will clash. A solution for this is using [CSS Modules.](https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/) But before that, you can have a read here about the [problem with global css](https://css-tricks.com/css-modules-part-1-need/)